Examples/GetFields/show_fields.py

```diff
@@ -0,0 +1,31 @@
+############################################################
+# Step 1)  Use Datasource object from the Document API
+############################################################
+from tableaudocumentapi import Datasource
+
+############################################################
+# Step 2)  Open the .tds we want to inspect
+############################################################
+sourceTDS = Datasource.from_file('World.tds')
+
+############################################################
+# Step 3)  Print out all of the fields and what type they are
+############################################################
+print('----------------------------------------------------------')
+print('--- {} total fields in this datasource'.format(len(sourceTDS.fields)))
+print('----------------------------------------------------------')
+count = 0
+for field_key, field in sourceTDS.fields.items():
+    count += 1
+    print('{:>4}: {} is a {}'.format(count, field.name, field.datatype))
+    blank_line = False
```

Comment:
If you always want the extra newline, you can pass end='\n\n' to the print function, and remove the blank_line variable.

If I understand the intent correctly